test-core.ini

```diff
@@ -31,6 +31,9 @@ ckan.datapusher.formats = csv xls xlsx tsv application/csv application/vnd.ms-ex
 ## Solr support
 solr_url = http://127.0.0.1:8983/solr/ckan
 
+# Redis URL. Use a separate Redis database for testing.
+redis_url = redis://localhost:6379/1
```

Comment:
I chose the name similar to `solr_url`, but namespacing it could make sense. However, I'd go with something like `ckan.redis.url` so that we can add more Redis-related settings later on if the need arises. I'd rather use URL than URI, although both would be correct here: All the other config options use URL, so let's keep that consistent.